_includes/releases/cloud/2023-05-15.md

```diff
@@ -0,0 +1,16 @@
+## May 15, 2023
+
+In addition to many of the Feature Highlights in the [CockroachDB v23.1.0 Release Notes](v23.1.html), the following features are specifically available to Cloud clusters once they are upgraded:
+
+<h3> General changes </h3>
+
+- {{ site.data.products.serverless }} multi-region is now publicly available in [preview](../{{site.versions["stable"]}}/cockroachdb-feature-availability.html#feature-availability-phases). Multi-region benefits like reduced latency and high availability are now achievable in conjunction with the ease-of-operation and automatic scaling advantages of the {{ site.data.products.serverless }} deployment model.
+    
+    - [Regions available](../cockroachcloud/serverless-faqs.html#what-regions-are-available-for-cockroachdb-serverless-clusters): 6 regions in AWS and 6 regions in GCP.
+    - A primary region for the cluster is specified upon creation.
+    - The cluster’s regional configuration applies to all databases by default, so it is not necessary to run [`ALTER DATABASE`](../{{site.versions["stable"]}}/alter-database.html) to configure regions when adding a database to the cluster.
```

Comment:
Suggest editing this link slightly so the link text is `ALTER DATABASE ... ADD REGION` and the link target is `v23.1/alter-database.html#add-region`